mono/metadata/filewatcher.c

```diff
@@ -113,21 +114,20 @@ typedef struct FAMEvent {
 	gint code;
 } FAMEvent;
 
-gboolean
+MonoBoolean
```

Comment:
This no longer matches the C# side.
I agree it is a good change, but they need to match, I think, in case the upper bits are not defined by codegen, and then you have to update monolite.

```
Searching for 'InternalFAMNextEvent'...
C:\s\mono2\mcs\class\System\System.IO\FAMWatcher.cs(270):					if (InternalFAMNextEvent (ref conn, out filename,
C:\s\mono2\mcs\class\System\System.IO\FAMWatcher.cs(452):		extern static int InternalFAMNextEvent (ref FAMConnection fc, out string filename,
2 occurrence(s) have been found.
```